File: golang/gcd.go
package main
import(
	"fmt"
)


func abigi(a, b int)(int){
	if a<b {
	   a, b = b, a
	}
	if a % b == 0{
		return b
	}else{
		num:= abigi(b, a%b)
		return num
		
	}
	return 0
}


func main(){
	var number1, number2 int
	fmt.Printf("Enter the first number")	
	fmt.Scanln(&number1)
	fmt.Printf("Enter the second number")
	fmt.Scanln(&number2)
	GCD:=abigi(number1, number2)
	fmt.Println("GCD of ",number1," and ",number2,"is",GCD)
}

File: Python/GCD.py
def gcd(a,b):

    small=0

    gd=0

    if a>b:

        small==b

    else:

        small==a

    for i in range(1, small+1):

        if((a % i == 0) and (b % i == 0)):

            gd=i

    return gd

a=int(input("Enter the first number: "))

b=int(input("Enter second number: "))

t=gcd(a,b)

print("GCD is:",t)
